content/doc/administration/requirements/java.adoc

```diff
@@ -11,6 +11,9 @@ Java 8 or Java 11 are required for running modern versions of Jenkins. Upgrading
 
 Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"]
 
+Starting with the weekly release line 2.353, Java 11 or Java 17 will be required to use Jenkins.
+Java 8 will still be supported in the Long Term Support or LTS release line, but the LTS release line's requirements will eventually mirror the weekly release line requirements.
```

Comment:
```suggestion
Java 8 will still be supported in the long-term support (LTS) release line, and the LTS release line's requirements will eventually mirror the weekly release line requirements.
```

Reply:
Thanks @b-fisher-cb and @Dan-Heath I was able to update the wording and make it more consistent with the LTS page.

The only thing that I kept was the "but" so that it can better explain the transition in potential translations